install_ROS.sh

```diff
@@ -12,9 +12,9 @@ echo 'deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://ap
 sudo apt-get update
 sudo rm /usr/share/keyrings/kitware-archive-keyring.gpg
 sudo apt-get install kitware-archive-keyring
```

Comment:
Are these still needed?

Reply:
I'm not sure the bug is still in the package, but I've changed the script to use the official install script from kitware.